src/test/kotlin/uk/gov/justice/digital/hmpps/prisonertonomisupdate/helpers/Messages.kt

```diff
@@ -68,6 +40,9 @@ fun retryMessage() = """
       }
 """.trimIndent()
 
+fun incentiveMessagePayload(incentiveId: Long) =
+  """{"eventType":"incentives.iep-review.inserted", "additionalInformation": {"id":"$incentiveId"}}"""
+
 fun incentiveCreatedMessage(incentiveId: Long) = """
```

Comment:
Does this one go also - or is it still being used?

Reply:
still used by unit test